.gitignore

```diff
@@ -4,8 +4,8 @@ __pycache__
 .env
 .idea
 .pytest_cache
+.venv
 get-pip.py
```

Comment:
I'm also ignoring the entire venv file - maybe we should look at this as a group?

Reply:
We can ignore them both, sorry for removing it.

Reply:
I added it back